modules/configuration/src/main/java/org/apache/ignite/internal/configuration/util/ConfigurationNotificationsUtil.java

```diff
@@ -622,7 +622,7 @@ public Void visitNamedListNode(String key, NamedListNode<?> newNamedList) {
 
                 for (String name : newNamedList.namedListKeys()) {
                     DynamicConfiguration<InnerNode, ?> newNodeCfg =
-                            (DynamicConfiguration<InnerNode, ?>) namedDynamicConfig(cfgNode, key).get(name);
+                            (DynamicConfiguration<InnerNode, ?>) namedDynamicConfig(cfgNode, key).touchMembers().get(name);
```

Comment:
I don't think you should do it on every iteration

Reply:
You are right, corrected it.